### PR TITLE
Updated Lesson 1 Example Target

### PR DESCRIPTION
**Pull request by emmalieb**
Found new target for demonstration. Need input on FFI to light curve part, looks strange to me and cannot confirm why. Most time this week spent trying to find a good example, if this one still isn't good then I'll need some help, thanks!

### CODE REVIEW

File: notebooks/01-lightcurves/01-Lightcurves.ipynb [hunk 1]
```diff
@@ -56,9 +56,11 @@
    "source": [
```
Comment (ttdu) [inline notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>ok, last round of feedback. these items are relatively minor and then we should be good to go!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (ttdu) [inline notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <pre class="ql-syntax" spellcheck="false">Line #1.    #We can get an exact id from our table we quieried above
</pre><p> </p><p>queried</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (ttdu) [inline notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <blockquote>am HDUlist</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (ttdu) [inline notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>I think we can keep this example, and have the takeaway be "you actually can't see the transits here"</p><p> </p><p>If you look closely, you'll notice two things:</p><ol><li>The amplitude of the oscillations increases over time</li><li>The period is roughly 1 day</li></ol><p> </p><p>This is due to stray light from Earth entering the field of view -- the day-period oscillation is actually Earth itself turning! I think that's a cool lesson to share, and it saves us needing to do further analysis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (ttdu) [inline notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>thoughts on removing this homework exercise, given the difficulty of background subtraction? I think it's ok for a "simpler" lesson like this to omit hw</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (emmalieb) [reply notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>commented it out</p>

Comment (emmalieb) [reply notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>changed the previous text and added most of this wording and mentioned to stay tuned for the next lesson on data quality</p>

Comment (emmalieb) [reply notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>fixed!</p>

Comment (emmalieb) [reply notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>fixed!</p>

Comment (emmalieb) [reply notebooks/01-lightcurves/01-Lightcurves.ipynb:56]: <p>great!</p>

**Review by ttdu — APPROVED** [review]

**Pull request merged**